src/test/java/trello/ui/main.java

```diff
@@ -5,41 +5,84 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.junit.Assert;
 
 import java.util.List;
 
 public class main {
+
+    private static void waitForPageLoaded(WebDriver driver, long sleep) {
+        ExpectedCondition<Boolean> expectation = new
+                ExpectedCondition<Boolean>() {
+                    public Boolean apply(WebDriver driver) {
+                        return ((JavascriptExecutor) driver).executeScript("return document.readyState").toString().equals("complete");
+                    }
+                };
+        try {
+//            Thread.sleep(1000);
+            Thread.sleep(sleep);
+            WebDriverWait wait = new WebDriverWait(driver, 30);
+            wait.until(expectation);
+        } catch (Throwable error) {
+            Assert.fail("Timeout waiting for Page Load Request to complete.");
+        }
+    }
+
     public static void main(String[] args){
-        //Configure driver manually
-//        System.setProperty("webdriver.chrome.driver", "d:\\otros\\webUI\\chromedriver_win32\\chromedriver.exe");
+
         //Configure web driver manager
         WebDriverManager.chromedriver().setup();
 
-        //Create web driver
-        //Open browser
-        WebDriver driver = new ChromeDriver();
-        //Navigate
-//        driver.get("https://www.ebay.com/");
-        driver.navigate().to("https://www.ebay.com/");
-        //Close browser
-//        driver.close();
-        //Find element
-        WebElement searchBox = driver.findElement(By.cssSelector("#gh-ac"));
-        searchBox.sendKeys("Cellphones");
-        searchBox.clear();
-        searchBox.sendKeys("Laptops");
-        //Click search
-        WebElement searchButton = driver.findElement(By.cssSelector("#gh-btn"));
-        searchButton.click();
-        //Close browser and kill process
+        //Create web driver. Open browser
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--lang=en");
```

Comment:
@ArturoAmoroso I guess the --lang option required because of some locator with english text, my suggestion is avoiding text except for validations, you could use some xpath axis if were necessary to avoid this.